Makefile

```diff
@@ -88,25 +75,19 @@ update-notices:
 ### Local testing targets. ###
 .PHONY: test
 test:
-	for TEST in $(TEST_PKGS) ; do \
-		go test $(BUILD_TAGS) -timeout $(TEST_TIMEOUT)s $$TEST ; \
-	done
+	go test $(BUILD_TAGS) -timeout $(TEST_TIMEOUT)s -p 1 ./...
```

Comment:
Here is my understanding of Go's test/build parallelism:

`go test ./...` and `go build ./...` will by default test and build all packages in the directory in parallel with `GOMAXPROCS` (set to the number of cpu cores on the machine by default) respectively. In other words, those commands are by default run with `-p GOMAXPROCS`. `-p 1` allows testing/building with `./...` but runs packages synchronously. See my comment on GODRIVER-2093 as to why setting `-p` higher than 1 for `go test` is perhaps not feasible at the moment.

Calling `t.Parallel()` in a test, on the other hand, specifies whether that test should be run in parallel with other tests that call `t.Parallel()`. `go test` is by default called with `-parallel GOMAXPROCS`, indicating that tests marked with `t.Parallel()` can run in parallel on `GOMAXPROCS` cores. `-parallel` and `-p` are distinct.